core/ordersync/ordersync.go

```diff
@@ -39,6 +39,12 @@ const (
 	maxRequestsPerSecond = 30
 	// requestsBurst is the maximum number of requests to allow at once.
 	requestsBurst = 10
+	// ordersyncJitterAmount is the amount of random jitter to add to the delay before
+	// each run of ordersync in PeriodicallyGetOrders. It is bound by:
+	//
+	//    (approxDelay - approxDelay * jitter) <= actualDelay <= (approxDelay + approxDelay * jitter)
```

Comment:
I don't feel strongly about this suggestion, but I think this could be a bit clearer if it is written:

```
approxDelay * (1 - jitter) <= actualDelay <= approxDelay * (1 + jitter)
```

My brain pattern matches this faster because it's what I would expect to see, but it's simple, so I'll leave this to your discretion.

Reply:
I like this better too. Just updated it.